newsfragments/705.feature

```diff
@@ -0,0 +1 @@
+An image template with a single ``#`` is expanded to match non-zero padded sequential numbers.
```

Comment:
_A single_ `#`

Worth mentioning in the change log that to open files with _exactly one_ digit we recommend `dials.import prefix_?.img` or similar?